### PR TITLE
Find some duplicate logic in LMouseListener and PPMouse #464

### LINKED ISSUE #464: Find some duplicate logic in LMouseListener and PPMouse
@KaranKamath I found these two classes are similar, are you able to merge them together?

### PR DESCRIPTION
**Pull request by digawp**
Fixes #464 
- Tested both double click and mouse up events; both are detected and working correctly.
- I took the liberty to remove some unused code.

### CODE REVIEW

**Comment by damithc** [comment]
Your PR names don't seem to follow the convention used by the project.

**Comment by digawp** [comment]
PR name changed.

File: PowerPointLabs/PowerPointLabs/ColorPicker/ColorPane.cs [hunk 1]
```diff
@@ -392,9 +389,7 @@ private void BeginEyedropping()
 
             _timerCounter = 0;
             timer1.Start();
-            _native = new LMouseUpListener();
-            _native.LButtonUpClicked +=
-                new EventHandler<SysMouseEventInfo>(_native_LButtonClicked);
+            PPMouse.LButtonUpClicked += _native_LButtonClicked;
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/ColorPicker/ColorPane.cs]: pls help to rename _native_LButtonClicked to something LeftButtonClickHandler

File: PowerPointLabs/PowerPointLabs/PPMouse.cs [hunk 2]
```diff
@@ -49,9 +49,11 @@ private static bool IsHookSuccessful()
 
         //Delegate
         public delegate void DoubleClickEventDelegate(PowerPoint.Selection selection);
+        public delegate void LButtonUpClickedDelegate();
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/PPMouse.cs]: pls rename this with full name, e.g. LeftButton

File: PowerPointLabs/PowerPointLabs/PPMouse.cs [hunk 3]
```diff
@@ -49,9 +49,11 @@ private static bool IsHookSuccessful()
 
         //Delegate
         public delegate void DoubleClickEventDelegate(PowerPoint.Selection selection);
+        public delegate void LButtonUpClickedDelegate();
 
         //Handler
         public static event DoubleClickEventDelegate DoubleClick;
+        public static event LButtonUpClickedDelegate LButtonUpClicked;
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/PPMouse.cs]: Also rename this with full name

**Pull request merged**